lib/Facebook.js

```diff
@@ -851,6 +909,11 @@ function Facebookbot(configuration) {
         return 'sha1=' + hmac.digest('hex');
     }
 
+    function getAppSecretProof(dataToHash, key) {
```

Comment:
I'm probably missing the big picture, but is this method intended to be used somewhere else?

If not, I suggest to use more explicit parameter names, like:

```
function getAppSecretProof(access_token, app_secret) {
    var hmac = crypto.createHmac('sha256', app_secret);
    return hmac.update(access_token).digest('hex');
}
```

Reply:
totally agree 👍